### DISCUSSION TITLE
How to set up a view that show TODO scheduled for today and earlier via org-ql-view

### DISCUSSION THREAD

**Discussion by capfredf** [alphapapa/org-ql#449] [Q&A] [answered]
Hi, there. Thanks for the wonderful package! I am trying to replace my old org-agenda using org-ql. I wanted to set up a view to show all my tasks scheduled for today and earlier. How do I do that?

**Comment by alphapapa** [comment, marked as answer]
Hi, glad you're enjoying it.  Can you be a bit more specific with your question?  It should generally be covered by the documentation, e.g. see the `scheduled` and `todo` predicates.

**Comment by capfredf** [comment]
Thanks for the pointer. I wasn't clear. My question was if there is a way to a customized view for org-ql-view to select?

A new question that just came to mind is I kind of want to a logical or predicate, but it does seem that org-ql provides one. Or is there a way to combine results from two org-ql-search. For example, I can show entries under a heading named AA in my agenda file with one org-ql-search. Ditto for all the scheduled entries in the agenda file. But I haven't wrapped my head around how to show them together.

**Reply by capfredf** [reply]
NVM. I missed the example.org. I've found what I wanted
